profile/ucs_server_profile/terraform/ucs_server_profile.tf

```diff
@@ -1,17 +1,148 @@
-resource "intersight_server_profile" "server1" {
-  name   = "server1"
-  action = "No-op"
+provider "intersight" {
+  endpoint        = "https://intersight.com"
+  apikey          = "xxxxx27564612d30dxxxxx/5f21c9d97564612d30dd575a/5f9a8b877564612xxxxxxxx"
+  secretkey       = "C:\\secretKey.txt"
+}
+
+resource "intersight_ntp_policy" "ntp_policy" {
+  name        = "ntp1"
+  description = "test policy"
+  enabled     = true
+  ntp_servers = ["ntp.esl.cisco.com", "time-a-g.nist.gov", "time-b-g.nist.gov"]
+
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+# Create SMTP policy
+resource "intersight_smtp_policy" "smtp_policy" {
+  name             = "smtp1"
+  description      = "testing smtp policy"
+  enabled          = false
+  smtp_port        = 32
+  min_severity     = "critical"
+  smtp_server      = "10.10.10.1"
+  sender_email     = "IMCSQAAutomation@cisco.com"
+  smtp_recipients  = ["aw@cisco.com", "cy@cisco.com", "dz@cisco.com"]
+
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+# Create SNMP policy
+resource "intersight_snmp_policy" "snmp_policy" {
+  name                    = "snmp1"
+  description             = "testing smtp policy"
+  enabled                 = true
+  snmp_port               = 1983
+  access_community_string = "dummy123"
+  community_access        = "Disabled"
+  trap_community          = "TrapCommunity"
+  sys_contact             = "xyz"
+  sys_location            = "SJC"
+  engine_id               = "abc"
+
+  snmp_users {
+    name           = "demouser"
+    privacy_type   = "AES"
+    security_level = "AuthPriv"
+    auth_type      = "SHA"
+    auth_password  = "dummyPassword"
+    privacy_password = "dummyPrivatePassword"
+  }
+
+  snmp_traps {
+    destination = "10.10.10.1"
+    enabled     = false
+    type        = "Trap"
+    user        = "demouser"
+    port        = 660
+    object_type = "SnmpTrap"
+  }
+
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+# Create a server profile template
```

Comment:
what is the use of server profile template creation here ?

Reply:
Removed and addressed.